javascript/README.md

```diff
@@ -382,3 +382,21 @@ if ("0" == false) {console.log('foo')} // this will console log! bad!
 // good
 if ("0" === false) {console.log('foo')}
 ```
+
+##Calling array methods on array-like objects
+
+Use functions on Array.prototype for array-like objects. (eg. 'arguments' is not an array, but 
+is array-like).
+
+###Prefer using methods from `[]` over `Array.prototype`
+
+Performance between the two styles is almost identical. Prefer the short, more succinct version.
+
+```javascript
+// Good
+[].slice.call(arguments); 
```

Comment:
Just tried it, and that doesn't seem to work.  I believe the correct syntax is to pass `arguments` as the first call, as it's "array-like".
```
> var x = function() {
... var y = [].slice.call(arguments);
... console.log("Result: " + y);
... }
undefined
> x(29392,2,2,2)
Result: 29392,2,2,2
undefined
> var x = function() {
... var y = [].slice.call(null, arguments);
... console.log("Result: " + y);
... }
undefined
> x(2,3,4)
TypeError: Array.prototype.slice called on null or undefined
    at x (repl:2:18)
    at repl:1:1
    at REPLServer.defaultEval (repl.js:164:27)
    at bound (domain.js:250:14)
    at REPLServer.runBound [as eval] (domain.js:263:12)
    at REPLServer.<anonymous> (repl.js:393:12)
    at emitOne (events.js:82:20)
    at REPLServer.emit (events.js:169:7)
    at REPLServer.Interface._onLine (readline.js:210:10)
    at REPLServer.Interface._line (readline.js:549:8)
```